src/MaxText/__init__.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 __author__ = "Google LLC"
-__version__ = "2025.09.03"
+__version__ = "2025.09.08"
```

Comment:
How often is this supposed to be updated?

Reply:
I try and remember to update it, no great issue until we have proper python versioning which is coming shortly. We can alternatively infer it from git and generate it on merge to `main` + copybara